plugins/env.go

```diff
@@ -0,0 +1,31 @@
+package plugins
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-envparse"
+)
+
+// ParseEnvFile returns a slice of key/value pairs parsed from the file at filepath.
+// As a special case, empty filepath returns nil without error.
+func ParseEnvFile(filepath string) ([]string, error) {
```

Comment:
I'm thinking through if this should live in `chainlink-common`, but it's probably best here.